generators/client/files-angular.js

```diff
@@ -58,6 +58,13 @@ const files = {
                 'content/css/_vendor.css',
                 'content/css/_documentation.css'
             ]
+        },
+        {
+            condition: generator => !generator.useSass && generator.enableI18nRTL,
```

Comment:
Does it mean that enableI18nRTL is a global option for the whole app? 
If so, can we build an app with both a mix of LTR and RTL languages?